servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpServerBuilder.java

```diff
@@ -268,13 +269,15 @@ public Single<HttpServerContext> listenStreaming(final StreamingHttpService serv
     @Override
     public Single<HttpServerContext> listenBlocking(final BlockingHttpService service) {
         HttpExecutionStrategy executionStrategy = computeServiceStrategy(BlockingHttpService.class, service);
-        return listenForService(toStreamingHttpService(executionStrategy, service), executionStrategy);
+        StreamingHttpService streamingService = toStreamingHttpService(executionStrategy, service);
+        return listenForService(streamingService, streamingService.requiredOffloads());
     }
 
     @Override
     public Single<HttpServerContext> listenBlockingStreaming(final BlockingStreamingHttpService service) {
         HttpExecutionStrategy executionStrategy = computeServiceStrategy(BlockingStreamingHttpService.class, service);
```

Comment:
Since the `executionStrategy` is used only once, consider inlining it instead of using a local variable.